suite-native/module-accounts/src/screens/AccountsScreen.tsx

```diff
@@ -1,15 +1,93 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+import { View } from 'react-native';
 
-import { Screen } from '@suite-native/navigation';
-import { Box, Text } from '@suite-native/atoms';
+import { useFocusEffect } from '@react-navigation/native';
+
+import {
+    AccountsStackParamList,
+    AccountsStackRoutes,
+    Screen,
+    StackProps,
+} from '@suite-native/navigation';
+import { Box, Button, Chip } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { CryptoIcon } from '@trezor/icons';
+import { NetworkSymbol } from '@suite-common/wallet-config';
 
 import { AccountsList } from '../components/AccountsList';
 
-export const AccountsScreen = () => (
-    <Screen>
-        <Box marginBottom="medium">
-            <Text color="gray600">All Accounts</Text>
-        </Box>
-        <AccountsList />
-    </Screen>
-);
+const assetsFilterStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    borderRadius: utils.borders.radii.round,
+    marginBottom: utils.spacings.medium,
+}));
+
+const assetsFilterItemStyle = prepareNativeStyle(utils => ({
+    marginRight: utils.spacings.small,
+}));
+
+const discoverAssetsButtonStyle = prepareNativeStyle(utils => ({
+    borderRadius: utils.borders.radii.round,
+}));
+
+export const AccountsScreen = ({
+    route,
+}: StackProps<AccountsStackParamList, AccountsStackRoutes.Accounts>) => {
+    const { applyStyle } = useNativeStyles();
+    const [selectedAssets, setSelectedAssets] = useState<NetworkSymbol[]>([]);
+
+    const { currencySymbol } = route.params;
+
+    useFocusEffect(
+        useCallback(() => {
```

Comment:
can you throw it to this callback to variable outside of `useFocusEffect`? Something like `handleScreenFocus` or smmthing like that would be nice. Also please move `const { currencySymbol } = route.params;` inside of it because it's used only there

Reply:
Sure - [3b74ca6b24ac779493213183de00588f27fbf0b6](https://github.com/trezor/trezor-suite/pull/6311/commits/dd3b60abc9fa52d44e936115d3b9dbbf7fea2ab3)